client/data/site-profiler/types.ts

```diff
@@ -187,6 +187,7 @@ export interface PerformanceMetricsItemQueryResponse {
 	type: 'warning' | 'fail';
 	displayValue?: string;
 	details?: PerformanceMetricsDetailsQueryResponse;
+	metricSavings: { FCP: number; LCP: number };
```

Comment:
The metricSavings can be FCP, LCP, CLS, or INP, and all of them are optional. Don't you want to add them?